messaging.fml.yaml

```diff
@@ -0,0 +1,211 @@
+            USER_RECENTLY_INSTALLED: days_since_install < 7
+            USER_RECENTLY_UPDATED: days_since_update < 7 && days_since_install != days_since_update
+            USER_TIER_ONE_COUNTRY: ('US' in locale || 'GB' in locale || 'CA' in locale || 'DE' in locale || 'FR' in locale)
+            USER_EN_SPEAKER: "'en' in locale"
+            USER_DE_SPEAKER: "'de' in locale"
+            USER_FR_SPEAKER: "'fr' in locale"
+            DEVICE_ANDROID: os == 'Android'
+            DEVICE_IOS: os == 'iOS'
+            ALWAYS: "true"
+            NEVER: "false"
+            I_AM_DEFAULT_BROWSER: "is_default_browser_string == 'true'"
+            I_AM_NOT_DEFAULT_BROWSER: "is_default_browser_string == 'false'"
+            USER_ESTABLISHED_INSTALL: "number_of_app_launches >=4"
+          actions:
+            ENABLE_PRIVATE_BROWSING: ://enable_private_browsing
+            INSTALL_SEARCH_WIDGET: ://install_search_widget
+            MAKE_DEFAULT_BROWSER: ://make_default_browser
+            VIEW_BOOKMARKS: ://urls_bookmarks
+            VIEW_COLLECTIONS: ://home_collections
+            VIEW_HISTORY: ://urls_history
+            VIEW_HOMESCREEN: ://home
+            OPEN_SETTINGS_ACCESSIBILITY: ://settings_accessibility
+            OPEN_SETTINGS_ADDON_MANAGER: ://settings_addon_manager
+            OPEN_SETTINGS_DELETE_BROWSING_DATA: ://settings_delete_browsing_data
+            OPEN_SETTINGS_LOGINS: ://settings_logins
+            OPEN_SETTINGS_NOTIFICATIONS: ://settings_notifications
+            OPEN_SETTINGS_PRIVACY: ://settings_privacy
+            OPEN_SETTINGS_SEARCH_ENGINE: ://settings_search_engine
+            OPEN_SETTINGS_TRACKING_PROTECTION: ://settings_tracking_protection
+            OPEN_SETTINGS_WALLPAPERS: ://settings_wallpapers
+            OPEN_SETTINGS: ://settings
+            TURN_ON_SYNC: ://turn_on_sync
+          styles:
+            DEFAULT:
+              priority: 50
+              max-display-count: 5
+            SURVEY:
+              priority: 55
+              max-display-count: 10
+            PERSISTENT:
+              priority: 50
+              max-display-count: 20
+            WARNING:
+              priority: 60
+              max-display-count: 10
+            URGENT:
+              priority: 100
+              max-display-count: 10
+          messages:
+            default-browser:
+              text: default_browser_experiment_card_text
+              action: "MAKE_DEFAULT_BROWSER"
+              trigger: [ "I_AM_NOT_DEFAULT_BROWSER","USER_ESTABLISHED_INSTALL" ]
+              style: "PERSISTENT"
+              button-label: preferences_set_as_default_browser
+
+      - channel: developer
+        value:
+          styles:
+            DEFAULT:
+              priority: 50
+              max-display-count: 100
+            EXPIRES_QUICKLY:
+              priority: 100
+              max-display-count: 1
+          notification-config:
+            polling-interval: 180 # 3 minutes
+
+objects:
```

Comment:
out of interest - the original file had this in a `types:` tag. What was that doing and is it no longer required?

Reply:
The `types` tag turned out to be unnecessary cruft, and we quietly deprecated it.

Reply:
Gotcha, thanks 👍